cadc-download-manager/src/main/java/ca/nrc/cadc/dlm/handlers/DataLinkServiceResolver.java

```diff
@@ -67,75 +67,75 @@
 
 package ca.nrc.cadc.dlm.handlers;
 
-
+import java.io.FileNotFoundException;
+import java.io.FileReader;
 import java.net.URI;
-import java.net.URL;
 import java.util.Properties;
 import org.apache.log4j.Logger;
 
 /**
  * Class to find the right DataLink service resourceIdentifier to use.
- * 
+ *
  * @author pdowler
  */
-public class DataLinkServiceResolver 
-{
-    private static final Logger log = Logger.getLogger(DataLinkServiceResolver.class);
-
+public class DataLinkServiceResolver {
     public static final String DEFAULT_KEY = DataLinkServiceResolver.class.getName() + ".resourceID";
-    
+    private static final Logger log = Logger.getLogger(DataLinkServiceResolver.class);
+    private static final String RESOURCEID_DEFAULT =  "ivo://cadc.nrc.ca/caom2ops";
+    private static final String RESOURCEID_KEY = "ca.nrc.cadc.dlm.handlers.DataLinkServiceResolver.resourceID";
```

Comment:
We have both DEFAULT_KEY and RESOURCEID_KEY, which are the same value.  Pick one.